packages/core/src/browser/common-frontend-contribution.ts

```diff
@@ -149,6 +149,21 @@ export namespace CommonCommands {
         category: VIEW_CATEGORY,
         label: 'Close All Tabs'
     };
+    export const CLOSE_EDITOR: Command = {
+        id: 'core.close.editor',
+        category: VIEW_CATEGORY,
+        label: 'Close Editor'
```

Comment:
had a chat with @svenefftinge, he would prefer to name it like `Close Tab in Main Area` to avoid confusion around what is editor and what not